components/component/example.tsx

```diff
@@ -112,9 +119,11 @@ const Test: FunctionComponent<Props> = ({ name, item, spacing }) => {
         </div>
 
         <div className={view ? 'hidden' : 'block'}>
-          <pre className="p-4 overflow-auto h-[400px] lg:h-[600px] ring-2 ring-black rounded-lg">
-            <code className="language-html">{code}</code>
-          </pre>
+          {!view && (
```

Comment:
What's the benefit of this? 🤔

It's handled with the `className` above, feels like an extra step.